images/prow-tests/Dockerfile

```diff
@@ -71,8 +71,12 @@ RUN gem install mdl -v 0.5  # required because later version of mdl requires rub
 # Install our own tools
 RUN go get knative.dev/pkg/testutils/junithelper
 
+# And our scripts
+ENV SCRIPTS /scripts
+COPY scripts /scripts
+
 # Temporarily add test-infra to the image to build custom tools
-ADD . /go/src/knative.dev/test-infra
+COPY . /go/src/knative.dev/test-infra
```

Comment:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

"Although ADD and COPY are functionally similar, generally speaking, COPY is preferred. That’s because it’s more transparent than ADD."

Reply:
to preempt any questions :-D